opentelemetry/src/global/mod.rs

```diff
@@ -130,15 +130,13 @@
 //! [`MeterProvider`]: crate::metrics::MeterProvider
 //! [`set_meter_provider`]: crate::global::set_meter_provider
 
-mod error_handler;
```

Comment:
Could we also remove the `error_handler.rs` file?

Reply:
This is already removed, and it's content moved to otel_sdk:

<img width="466" alt="{91AE70A0-90DE-4DAE-B955-8F24899D4532}" src="https://github.com/user-attachments/assets/22fe68c3-cc86-4566-811f-e9abaf35ae4c">

Reply:
Ah yes. My bad! It would render as move in the diff.